test/test_assets.js

```diff
@@ -983,6 +984,122 @@ describe("assets", function () {
           });
         });
       });
+
+      describe("cache keys", function() {
+        var rootDir = testutils.fixtureDirectory("app_assets");
+        var rootDir2 = testutils.fixtureDirectory("app_assets_odd_names");
+
+        it("Assets.cacheKey includes httpPrefix", function(done) {
+          var source1 = new AssetsSource(rootDir, {
+            httpPrefix: "foo",
+          });
+          var source2 = new AssetsSource(rootDir, {
+            httpPrefix: "foo",
+          });
+          var source3 = new AssetsSource(rootDir, {
+            httpPrefix: "bar",
+          });
+          assert.equal(source1.cacheKey(), source2.cacheKey());
+          assert.notEqual(source1.cacheKey(), source3.cacheKey());
+          done();
```

Comment:
In mocha, we should likely not use `done()`, two reasons:

1. sync tests, such as this don't require it. It can safely be dropped.
2. async tests, should preferrably rely on a promise being returned in the `it` `function` body, as it does error handling correctly (timed out tests do to test failures suck)

Reply:
ok, I'll remove the use of done() in all these places